imaids/utils.py

```diff
@@ -131,6 +133,27 @@ def cosine_function(z, bamp, freq, phase):
     return bamp*_np.cos(freq*z + phase)
 
 
+def get_beff_fit(gap_over_period, beff, br):
```

Comment:
Here, the function could have a more descriptive name too. This one seems very specific to me.